### DISCUSSION TITLE
Item cost and storage

### DISCUSSION THREAD

**Discussion by walrhe93** [samolego/TraderNPCs#14] [Q&A] [answered]
Hello Samolego,

Currently I’m running a PaperMC server with a couple of plugins. Now I would like to switch to fabric. 
Your mods Taterzens and TraderNPCs seem to be able to add the functionality I’m looking for.
The current plugin I use is [shopkeepers](https://dev.bukkit.org/projects/shopkeepers)
The TraderNPCs mod comes close, but has two differences (For me problems).

-	Setting up the trades uses items from your inventory
-	You have infinite trades without stock you need to refill.

I was wondering if it might be possible to change these options.

I look forward to hearing from you.
Thank you in advance

Kind regards,
Walrhe93

**Comment by samolego** [comment, marked as answer]
There's survival trader profession which fixes the second issue; first one is a feature request 😁.

**Reply by walrhe93** [reply]
Thank you very much for your quick response. 
I’ve tested it and it works close enough for me to start using the mod.😁

I would like to add two requests if possible.
The first is as mentioned in the question to change it so setting the trades (at least for the survival trader) does not cost you the items.
The second is to make change the interaction so it’s the same as the normal trader. (Right click opens the trades window, crouch + right click opens the current window)

**Reply by samolego** [reply]
Hint: once the trader is set up (& unselected), it defaults to trade gui on right click.
Could be reversed though
